esphome/components/ethernet/ethernet_component.h

```diff
@@ -26,6 +26,12 @@ struct ManualIP {
   IPAddress dns2;  ///< The second DNS server. 0.0.0.0 for default.
 };
 
+enum class EthernetComponentState {
+  Stopped,
+  Connecting,
+  Connected,
```

Comment:
```suggestion
  STOPPED,
  CONNECTING,
  CONNECTED,
```

In ESPHome we use UPPER_SNAKE_CASE for enum constants (I think `clang-tidy` didn't detect it automatically here because it skips all ESP32-only files)

Reply:
Got it, thanks for the fix here.